ksql-functional-tests/src/test/resources/query-validation-tests/timestampformat.json

```diff
@@ -12,7 +12,7 @@
       "name": "timestamp format",
       "statements": [
         "CREATE STREAM TEST (ID bigint, event_timestamp VARCHAR) WITH (kafka_topic='test_topic', value_format='JSON', timestamp='event_timestamp', timestamp_format='yyyy-MM-dd''T''HH:mm:ssX');",
-        "CREATE STREAM TS AS select id, stringtotimestamp(event_timestamp, 'yyyy-MM-dd''T''HH:mm:ssX') as ets from test;"
+        "CREATE STREAM TS WITH (timestamp='ets') AS select id, stringtotimestamp(event_timestamp, 'yyyy-MM-dd''T''HH:mm:ssX') as ets from test;"
```

Comment:
I added the WITH TIMESTAMP here to use the source timestamp as the output timestamp.